### PR TITLE
scroll to bottom when text is generated

### PR DESCRIPTION
**Pull request by davidkoski**
- also restore circleci

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,17 +43,12 @@ workflows:
...
         - not: << pipeline.parameters.nightly_build >>
         - not: << pipeline.parameters.weekly_build >>
     jobs:
-      - hold:
-          type: approval
-      - apple/authenticate:
-          context: pr-approval
-      - mac_build_and_test:
-          requires: [ hold ]
+      - mac_build_and_test
```
Comment (davidkoski) [inline .circleci/config.yml:51]: Restore changes that were done to try and get non-fork PRs to build but broke main.

Comment (davidkoski) [reply .circleci/config.yml:51]: OK, it looks like this works -- the caveat is you can't name your branch something like `dkoski/scroll`

Comment (awni) [reply .circleci/config.yml:51]: I see so `name1/name2` breaks the pattern (I guess that's true in our other repos as well).

File: Applications/LLMEval/ContentView.swift [hunk 2]
```diff
@@ -16,11 +16,21 @@ struct ContentView: View {
...
+                    Text(llm.output)
+                        .textSelection(.enabled)
+
+                        .onChange(of: llm.output) { _, _ in
+                            sp.scrollTo("bottom")
+                        }
+
+                    Spacer()
+                        .frame(width: 1, height: 1)
+                        .id("bottom")
```
[the remark this answers is not in the document]
Comment (awni) [reply Applications/LLMEval/ContentView.swift:32]: ❤️  !

**Review by awni — APPROVED** [review]
Thanks!!

**Pull request merged**